core/src/main/resources/hudson/logging/LogRecorderManager/levels.jelly

```diff
@@ -34,7 +34,7 @@ THE SOFTWARE.
       ${%Logger Configuration}
       <t:help iconSize="large" tooltip="${%Additional information on log levels}" href="${%url}" />
     </h1>
-    <table class="sortable pane jenkins-table" id="logLevels">
```

Comment:
Unrelated?

Reply:
It is related, without this change the help icon is not rendered correctly.